docs/TinyAra_Config_Variables.html

```diff
@@ -978,7 +978,7 @@ <h1>Table of Contents</h1>
 <li><a href="#CONFIG_BOARD_FOTA_SUPPORT">1.3.12 <code>CONFIG_BOARD_FOTA_SUPPORT</code>: Enable board support for FOTA</a></li>
 <li><a href="#CONFIG_BOARD_COREDUMP_FLASH">1.3.13 <code>CONFIG_BOARD_COREDUMP_FLASH</code>: Enable crashlog dump to flash</a></li>
 <li><a href="#CONFIG_BOARD_RAMDUMP_FLASH">1.3.14 <code>CONFIG_BOARD_RAMDUMP_FLASH</code>: Enable ramdump to flash</a></li>
-<li><a href="#CONFIG_BOARD_RAMDUMP_UART">1.3.15 <code>CONFIG_BOARD_RAMDUMP_UART</code>: Enable ramdump via UART</a></li>
+<li><a href="#CONFIG_BOARD_DUMP_UART">1.3.15 <code>CONFIG_BOARD_DUMP_UART</code>: Enable dumping of contents via UART</a></li>
```

Comment:
Enable dumping of ram and flash contents via UART?

Reply:
Updated